homeassistant/components/pulseaudio_loopback/switch.py

```diff
@@ -140,6 +145,8 @@ def get_module_idx(self, sink_name, source_name):
 class PALoopbackSwitch(SwitchDevice):
     """Representation the presence or absence of a PA loopback module."""
 
+    _state_updated = False
```

Comment:
Remove this line, introduce this property in the constructor by setting `self._available` to `False`
```suggestion
```